scripts/check_frontend_test_coverage_test.py

```diff
@@ -354,6 +354,69 @@ def test_fully_covered_filenames_is_not_sorted(self) -> None:
                     ],
                 )
 
+    def test_check_if_file_should_be_checked(self) -> None:
```

Comment:
Please follow our conventions for writing backend tests: https://github.com/oppia/oppia/wiki/Backend-tests#guidelines-for-writing-good-tests

Reply:
Thanks, Done.